webpack.config.js

```diff
@@ -18,7 +18,10 @@ module.exports = (_env, argv) => {
                 library: 'JitsiMeetJS',
                 libraryTarget: 'umd',
                 path: path.join(process.cwd(), 'dist', 'umd')
-            })
+            }),
+            performance: {
+                hints: false
```

Comment:
Why is this needed here?

Reply:
Don't think there's a reason, just remnants from the old PR, removed.